Dockerfile

```diff
@@ -19,4 +19,7 @@ ENV PORT 80
 ENV HEALTH_HTTP_PORT 8080
 ENV SERVER_NAME _
 
+# Required to use envsubst
+RUN apt-get update && apt-get install -y gettext-base
+
```

Comment:
for cache reasons, it is best to run this command as soon as possible, i.e. before line 14.